### PR TITLE
feat: add support for extraVolumes and extraVolumeMounts

### PR DESCRIPTION
**Pull request by BillKalaitzis**
Adds support for the generic `extraVolumes` and `extraVolumeMounts` directives on our helmchart.
The capabilities provided by them have come in handy on the following discussion items:
- https://github.com/netdata/netdata/discussions/13425
- https://github.com/netdata/netdata/discussions/13463

### CODE REVIEW

File: charts/netdata/values.yaml [hunk 1]
```diff
@@ -333,6 +337,19 @@ child:
     rooms: ""
     url: "https://app.netdata.cloud"
 
+  extraVolumeMounts: []
+    ## Additional volume mounts for netdata child
+    ## E.g to mount all disks under / to be monitored via the diskspace plugin
+    # - name: root
+    #   mountPath: /host
+    #   readOnly: true
```
Comment (ilyam8) [inline charts/netdata/values.yaml:345]: Default [mountPropagation](https://kubernetes.io/docs/concepts/storage/volumes/#mount-propagation) is None. I think it makes sense to set it to HostToContainer in this example (_E.g to mount all disks under / to be monitored via the diskspace plugin_) or at least mention the option. What do you think?

Comment (BillKalaitzis) [reply charts/netdata/values.yaml:345]: Hey @ilyam8, that's a good catch! I've appended the suggested mountProgagation option in here: https://github.com/netdata/helmchart/pull/294/commits/cce91f8891dcdadf125046f6558f2856784cc65d

**Comment by ilyam8** [comment]
@BillKalaitzis readme should be updated too 😄

**Comment by BillKalaitzis** [comment]
> @BillKalaitzis readme should be updated too 😄

Done 😄

**Review by ilyam8 — APPROVED** [review]

**Review by M4itee — APPROVED** [review]

**Pull request merged**